src/App.jsx

```diff
@@ -2,15 +2,27 @@ import "./App.css";
 
 import SideBar from "./components/SideBar/SideBar";
 import TabBar from "./components/TabBar/TabBar";
-import Experience from "./components/Experience";
-
+import AnimationsBadge from "./components/AnimationsBadge/AnimationsBadge";
+import AnimationsStatus from "./components/AnimationsStatus/AnimationsStatus";
 
 function App() {
   return (
     <>
-      <SideBar />
-      <TabBar />
-      <Experience />
+      <div className="relative h-screen ">
+        <SideBar />
+        <TabBar />
+
+        <div className="absolute bottom-8  w-full flex items-center">
```

Comment:
This overlays the link to the About page which can't be clicked anymore.

Reply:
is this in the mobile view?

Reply:
your right, thanks, its fixed now